tools/topology/CMakeLists.txt

```diff
@@ -1,19 +1,18 @@
 set(SOF_TOPOLOGY_BINARY_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}")
-set(SOF_ALSA_TOOLS_DIR "${SOF_ROOT_SOURCE_DIRECTORY}/../tools/bin")
-set(ALSATPLG_CMD "${SOF_ALSA_TOOLS_DIR}/alsatplg")
+
 
 function(alsatplg_version  OUT_STATUS  OUT_VERSION)
-	execute_process(COMMAND ${ALSATPLG_CMD} --version
+	execute_process(COMMAND alsatplg --version
```

Comment:
Commit message refers to alsactl but this reverts the whole thing. This will cause invalid tplgs to be created by our CI system... (as the system alsa-lib is too old).